src/api/hooks/wishlist.mock.ts

```diff
@@ -0,0 +1,62 @@
+import { rest } from 'msw';
+
+import { BASE_URL } from '../instance';
+import { getWishlistPath } from './useGetWishlist';
+
+export const wishlistMockHandler = [
+  rest.get(getWishlistPath(), (_, res, ctx) => {
+    return res(ctx.json(WISHLIST_RESPONSE_DATA));
+  }),
+  rest.delete(`${BASE_URL}/api/wishes/:id`, (req, res, ctx) => {
```

Comment:
```suggestion
  rest.delete(`${getWishlistPath()}/:id`, (req, res, ctx) => {
```
위와 같이 작성하면 좀 더 일관성 있는 코드가 될 것 같아요-!